src/lib/umd.index.ts

```diff
@@ -99,4 +99,14 @@ const SynapseComponents = {
   ForumSearch,
 }
 
-export { SynapseComponents, SynapseConstants, SynapseClient, SynapseContext }
+// Include the version in the build
+// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-var-requires, @typescript-eslint/no-unsafe-member-access
+const SynapseReactClientVersion = require('../../package.json').version
```

Comment:
I want to do this:
![image](https://user-images.githubusercontent.com/17580037/172930387-bba360ad-ddbb-4442-be0a-b5467bae78c8.png)